### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by steelec**
Installs cleanly, all examples run successfully.

### CODE REVIEW

**Comment by juhuntenburg** [comment]
Build passing now, checking docs and install

File: build.sh [hunk 1]
```diff
@@ -190,7 +190,21 @@ cp -rv build/nighresjava/ ../
...
+echo "  Local installation:"
+echo "    python3 -m pip install ."
+echo "" 
+echo "  Custom installation (e.g., servers or modulular systems) requires some pre-setup to satisfy setuptools:"
+echo "  Where you should use the path to your custom install directory (e.g., /opt/quarantine/nighres/${release}/install/)"
+echo "    mkdir -p /opt/quarantine/nighres/${release}/install/lib/python3.7/site-packages/"
+echo "    export PYTHONPATH=/opt/quarantine/nighres/${release}/install/lib/python3.7/site-packages/:\$PYTHONPATH"
+echo "    python3 setup.py install --prefix /opt/quarantine/nighres/${release}/install/"
+echo "==================================================================================================================="
+echo ""
```
Comment (juhuntenburg) [inline build.sh]: I would like to move this part to the docs - installation section. It isn't relevant for most people and clutters / confuses the output. 

echo "" 
echo "  Custom installation (e.g., servers or modulular systems) requires some pre-setup to satisfy setuptools:"
echo "  Where you should use the path to your custom install directory (e.g., /opt/quarantine/nighres/${release}/install/)"
echo "    mkdir -p /opt/quarantine/nighres/${release}/install/lib/python3.7/site-packages/"
echo "    export PYTHONPATH=/opt/quarantine/nighres/${release}/install/lib/python3.7/site-packages/:\$PYTHONPATH"
echo "    python3 setup.py install --prefix /opt/quarantine/nighres/${release}/install/"

**Comment by juhuntenburg** [comment]
@steelec I removed the long explanation from the build script, if you want I would suggest putting it in the docs installation trouble shooting part instead.

**Comment by juhuntenburg** [comment]
@piloubazin: The docstring of the multiscale vessel filter is lacking explanation of the outputs. Can you pull the newest version of this branch and add it?

File: build.sh (hunk 1 above)
Comment (steelec) [reply build.sh]: Ok yes, better there.

**Comment by steelec** [comment]
I re-tested after realizing that I hadn't run the new mesh functions in example_02 since they were added and identified an error with the new mesh inflation function which looks to be related to pandas (perhaps a version issue?)?

the `sep='\s*'` in the pandas read_csv calls is where it currently crashes for me, using pandas version: `0.24.2`

changing to `sep='\s+'` seems to solve the issue for me


```Surface inflation

Outputs will be saved to /home/remoteuser/stechr/Documents/code/nighres_testing/nighres_examples/cortical_depth_estimation/
Traceback (most recent call last):
  File "example_02_cortical_depth_estimation.py", line 148, in <module>
    output_dir=out_dir)
  File "/opt/quarantine/nighres/1.2/install/lib/python3.7/site-packages/nighres-1.2.0-py3.7.egg/nighres/surface/surface_inflation.py", line 83, in surface_inflation
    orig_mesh = load_mesh(surface_mesh)
  File "/opt/quarantine/nighres/1.2/install/lib/python3.7/site-packages/nighres-1.2.0-py3.7.egg/nighres/io/io_mesh.py", line 39, in load_mesh
    points, faces, data = _read_vtk(surf_mesh)
  File "/opt/quarantine/nighres/1.2/install/lib/python3.7/site-packages/nighres-1.2.0-py3.7.egg/nighres/io/io_mesh.py", line 380, in _read_vtk
    engine='python')
  File "/opt/quarantine/anaconda3/2018.12/install/lib/python3.7/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/opt/quarantine/anaconda3/2018.12/install/lib/python3.7/site-packages/pandas/io/parsers.py", line 435, in _read
    data = parser.read(nrows)
  File "/opt/quarantine/anaconda3/2018.12/install/lib/python3.7/site-packages/pandas/io/parsers.py", line 1139, in read
    ret = self._engine.read(nrows)
  File "/opt/quarantine/anaconda3/2018.12/install/lib/python3.7/site-packages/pandas/io/parsers.py", line 2415, in read
    alldata = self._rows_to_cols(content)
  File "/opt/quarantine/anaconda3/2018.12/install/lib/python3.7/site-packages/pandas/io/parsers.py", line 3066, in _rows_to_cols
    self._alert_malformed(msg, row_num + 1)
  File "/opt/quarantine/anaconda3/2018.12/install/lib/python3.7/site-packages/pandas/io/parsers.py", line 2835, in _alert_malformed
    raise ParserError(msg)
pandas.errors.ParserError: Expected 9 fields in line 258401, saw 10. Error could possibly be due to quotes being ignored when a multi-char delimiter is used.```

**Comment by piloubazin** [comment]
Multi scale vessel filter updated. For the meshes, the change works for me too, so I committed it as well. Ready to merge, I would say...

**Comment by juhuntenburg** [comment]
I agree!

**Pull request merged**